docs/actions-swift.md

```diff
@@ -36,11 +36,11 @@ An action is simply a top-level Swift function. For example, create a file calle
 
 ```swift
 func main(args: [String:Any]) -> [String:Any] {
-    if let name = args["name"] as? String {
-        return [ "greeting" : "Hello \(name)!" ]
-    } else {
-        return [ "greeting" : "Hello stranger!" ]
-    }
+  if let name = args["name"] as? String {
+    return [ "greeting" : "Hello \(name)!" ]
+  } else {
+    return [ "greeting" : "Hello stranger!" ]
+  }
```

Comment:
I think that the Swift source code should be indented 4 spaces as per [The Swift Programming Language](https://docs.swift.org/swift-book/) book.